aten/src/ATen/native/native_functions.yaml

```diff
@@ -8440,7 +8440,7 @@
   python_module: special
   variants: function
   dispatch:
-    CompositeExplicitAutograd: special_entr
```

Comment:
I think this was correct.
`special_entr` calls `special_entr_out` which specifies the implemented backend (entry below).

So if `special_entr_out` supports more backend, `special_entr` will implicitly support them as well.

Reply:
It does not.

```
Tensor special_entr(const Tensor& self) { return unary_op_impl_float(self, entr_stub); }
```

The `special_entr` directly calls the `entr_stub`, which only supports CPU/CUDA (because dispatch stub only supports CPU/CUDA).

Reply:
Ah right. Sorry got confused!